includes/forms/template.php

```diff
@@ -1117,12 +1117,12 @@ function give_get_register_fields( $form_id ) {
 					if( ! give_is_setting_enabled( $is_guest_checkout ) ) {
 						echo Give()->tooltips->render(
 							array(
-								'tag_content' => '<input type="checkbox" name="give_create_account" id="give-create-account-" class="give-input" checked disabled /><input type="hidden" name="give_create_account" value="on" />',
+								'tag_content' => '<input type="checkbox" name="give_create_account" value="on" id="give-create-account-" class="give-input give-disabled" checked />',
```

Comment:
@raftaar1191 Please disable this checkbox when `Guest Donations` are disabled as a donor can't must login to donate.

Reply:
As discussed with @raftaar1191 over call, it is the issue due to caching. Then tested in FireFox, Chrome and Safari to double-check

@ravinderk This PR is working fine for me.